### DISCUSSION TITLE
Cannot deploy on Google Cloud

### DISCUSSION THREAD

**Discussion by ollyde** [vercel/next.js#20472] [Help]
Running the following command ```rm -rf ./build && npm run build && y | gcloud app deploy --project cyberpunk-2077 app.yaml```

Get the following
```The directory [build/cache/next-babel-loader] has too many files```

I have to run with node 10 btw. Because Google flex environment doesn't support min_instances properly (slow demo's for sales)

Any ideas? We're stuck now. QA department supposed to be testing on Monday 🙈

**Comment by JosephChotard** [comment]
I'm getting an issue where 1 or 2 of the files in build/cache/next-babel-loader won't upload which causes the deploy to fail. Might be caused by the same root issue?
